src/contracts/YourContract.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.4.24;
```

Comment:
This isn't needed for displaying NFTs, is it? Can we remove it and all its references?